pkg/solana/logpoller/orm_test.go

```diff
@@ -183,16 +202,17 @@ func TestLogPollerFilters(t *testing.T) {
 func TestLogPollerLogs(t *testing.T) {
```

Comment:
Does this test pass for you? When I run it locally, it fails on line 227 while comparing `CreatedAt` timestamps:
```
require.Equal(t, log, dbLogs[0]
```
It looks like the problem is just that `CreatedAt` is empty when the initial "random log" is generated, and filled in by the db. I guess we can fix this by just setting it back to nil before comparing?